src/main/java/life/genny/qwandaq/attribute/EntityAttribute.java

```diff
@@ -627,13 +637,52 @@ public void setFeedback(String feedback) {
 
 	@PreUpdate
 	public void autocreateUpdate() {
+
+		if (getValueString() != null) {
+			convertToMinIOObject();
+		}
+
 		setUpdated(LocalDateTime.now(ZoneId.of("Z")));
 	}
 
 	@PrePersist
 	public void autocreateCreated() {
 		if (getCreated() == null)
 			setCreated(LocalDateTime.now(ZoneId.of("Z")));
+
+		if (getValueString() != null) {
+			convertToMinIOObject();
+		}
+
+	}
+
+	public void convertToMinIOObject() {
+		log.info("Converting to MinIO");
+		try {
+			int limit = 4 * 1024; // 4Kb
+			// logic to push to minIO if it is greater than certain size
+			byte[] data = valueString.getBytes(StandardCharsets.UTF_8);
+			if (data.length > limit) {
+				log.info("Greater Size");
+				String fileName = QwandaQConstant.MINIO_LAZY_PREFIX+ baseEntityCode + "-" + attributeCode;
+				File theDir = new File("file-uploads/");
+				if (!theDir.exists()) {
+					theDir.mkdirs();
+				}
+				String fileInfoName = "file-uploads/".concat(fileName);
```

Comment:
Should "file-uploads" be a configurable name? If not, at least extract this to a constant